sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -390,6 +390,7 @@ struct _pi_mem : _pi_object {
   char *MapHostPtr;
 
   // Flag to indicate that this memory is allocated in host memory
+  // if created with PI_MEM_FLAGS_HOST_PTR_ALLOC and/or integrated devices.
```

Comment:
NIT: Let's not list all the cases when memory is allocated on host, since that might change in future.